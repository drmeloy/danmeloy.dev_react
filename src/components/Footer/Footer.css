.Footer {
  padding-left: 6%;
  height: 4vh;
  display: flex;
  justify-content: center;
  align-items: center;
  font-family: 'Raleway', sans-serif;
  color: #ffffffcc;
  font-size: .5em;
  background: linear-gradient(135deg, rgba(66, 66, 66, 0.46) 0%, rgba(66, 66, 66, 0.46) 12.5%,rgba(68, 68, 68, 0.46) 12.5%, rgba(68, 68, 68, 0.46) 25%,rgba(71, 71, 71, 0.46) 25%, rgba(71, 71, 71, 0.46) 37.5%,rgba(73, 73, 73, 0.46) 37.5%, rgba(73, 73, 73, 0.46) 50%,rgba(75, 75, 75, 0.46) 50%, rgba(75, 75, 75, 0.46) 62.5%,rgba(77, 77, 77, 0.46) 62.5%, rgba(77, 77, 77, 0.46) 75%,rgba(80, 80, 80, 0.46) 75%, rgba(80, 80, 80, 0.46) 87.5%,rgba(82, 82, 82, 0.46) 87.5%, rgba(82, 82, 82, 0.46) 100%),linear-gradient(45deg, rgb(41, 41, 41) 0%, rgb(41, 41, 41) 12.5%,rgb(43, 43, 43) 12.5%, rgb(43, 43, 43) 25%,rgb(46, 46, 46) 25%, rgb(46, 46, 46) 37.5%,rgb(48, 48, 48) 37.5%, rgb(48, 48, 48) 50%,rgb(50, 50, 50) 50%, rgb(50, 50, 50) 62.5%,rgb(52, 52, 52) 62.5%, rgb(52, 52, 52) 75%,rgb(55, 55, 55) 75%, rgb(55, 55, 55) 87.5%,rgb(57, 57, 57) 87.5%, rgb(57, 57, 57) 100%);
}