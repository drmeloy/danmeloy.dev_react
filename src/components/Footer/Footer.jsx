import React from 'react';
import styles from './Footer.css';

export default function Footer(){
  return (
    <>
      <footer className={styles.Footer}>
      Website powered by React | Built by Dan Meloy | Last update: March 2020
      </footer>
    </>
  );
}
