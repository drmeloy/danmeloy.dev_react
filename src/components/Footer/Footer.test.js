import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
