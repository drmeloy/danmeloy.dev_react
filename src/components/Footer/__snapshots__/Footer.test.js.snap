// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<Fragment>
  <footer
    className="Footer"
  >
    Website powered by React | Built by Dan Meloy | Last update: March 2020
  </footer>
</Fragment>
`;
